modules/parquet/src/parquetjs/utils/schema-converter-util.ts

```diff
@@ -0,0 +1,47 @@
+import type {ParquetSchema} from '../schema/schema';
```

Comment:
Same here, let's move this from the `parquetjs` folder to the `lib` folder.

Reply:
Moved to `lib/convert-schema.ts`